ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -717,6 +717,17 @@ objects:
 
           terraform apply -auto-approve -no-color &
           wait "$!"
+          # find bootstrap ip
+
+          BOOTSTRAP_IP=$(terraform state show 'module.bootstrap.external.ip_address' | grep ip_address | awk '{print $3}')
```

Comment:
You have `jq` in this container; I'd prefer that over `grep` and `awk`.  But I like using `... state show ...` too.  Looks like new Terraforms support [`-json`][1] [since v0.12.0][2].  Can we bump [our current 0.11.11 pin][3] to pick up a newer Terraform and use that?

[1]: https://www.terraform.io/docs/commands/show.html#json-output
[2]: https://github.com/hashicorp/terraform/commit/126e5f337f15ff1fc75ce3f0901c9ae8bcdf0841#diff-082421f08b93b1575676332140ab532cR33
[3]: https://github.com/openshift/installer/blob/d9a9648cf2330d467cca9f2988846d031464125e/images/installer/Dockerfile.upi.ci#L29

Reply:
Ah, a bump to v0.12.0 is in-flight, but it's complicated (openshift/installer#2057).  We'll circle back to this once we get that landed.

/lgtm